chainer/functions/array/flatten.py

```diff
@@ -13,12 +13,34 @@ def backward(self, inputs, grads):
 
 
 def flatten(x):
-    """Flatten a given array.
+    """Flatten a given array into one dimension.
```

Comment:
Ok. I will add the document about 0-dim, but I think the case is exceptional, so I will add the document as a note.